tests/pipeline/test_blaze.py

```diff
@@ -1876,6 +1876,54 @@ def test_checkpoints_out_of_bounds(self):
         self._test_checkpoints(checkpoints)
 
 
+class LastInGroupSortingTestCase(BlazeToPipelineTestCase):
```

Comment:
Though I forgot that subclassing won't work here because it's trying to run all the other tests under this class.